store/src/db/migrations.rs

```diff
@@ -39,21 +39,35 @@ pub static MIGRATIONS: Lazy<Migrations> = Lazy::new(|| {
 
             PRIMARY KEY (account_id),
             CONSTRAINT fk_block_num FOREIGN KEY (block_num) REFERENCES block_headers (block_num),
-            CONSTRAINT accounts_block_num_is_u32 CHECK (block_num >= 0 AND block_num < 4294967296)
+            CONSTRAINT accounts_block_num_is_u32 CHECK (block_num BETWEEN 0 AND 0xFFFFFFFF)
+        ) STRICT, WITHOUT ROWID;
+
+        CREATE TABLE
+            account_details
+        (
+            account_id INTEGER NOT NULL,
+            nonce INTEGER NOT NULL,
+            vault BLOB NOT NULL,
+            storage BLOB NOT NULL,
+            code BLOB NOT NULL,
+
+            PRIMARY KEY (account_id),
+            CONSTRAINT account_details_nonce_non_negative CHECK (nonce >= 0),
+            CONSTRAINT fk_account_id FOREIGN KEY (account_id) REFERENCES accounts (account_id)
         ) STRICT, WITHOUT ROWID;
```

Comment:
I thought we were going to replace this with `details` field in the `accounts` table. Or did you decide to keep it as a separate table?

Reply:
@bobbinth yes, as we decided earlier, I will change DB structure and remove this table.